### ISSUE TITLE
A11y: Self-collapsing mode - Function cannot be performed by keyboard alone

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5046]
There is no way to perform the function using only the keyboard on the same screen or on a qualifying conforming alternate version (Chrome). [https://docs.deque.com/issue-help/1.0.0/en/keyboard-inaccessible|https://docs.deque.com/issue-help/1.0.0/en/keyboard-inaccessible|smart-link] 

In Safari with {{ctrl}} + {{opt}} + {{arrow right}}, I was able to navigate through all elements. 

!Screen Shot 2024-10-09 at 11.15.19 AM.png|width=994,height=756,alt="Screen Shot 2024-10-09 at 11.15.19 AM.png"!

Uploaded file element

!Screen Shot 2024-10-09 at 11.55.24 AM.png|width=938,height=544,alt="Screen Shot 2024-10-09 at 11.55.24 AM.png"!

!Screen Shot 2024-10-09 at 11.14.47 AM.png|width=988,height=686,alt="Screen Shot 2024-10-09 at 11.14.47 AM.png"!

!Screen Shot 2024-10-10 at 14.23.12 PM.png|width=944,height=498,alt="Screen Shot 2024-10-10 at 14.23.12 PM.png"!

*HTML editor:* buttons (Editor broken - [https://shopware.atlassian.net/browse/CUS-730|https://shopware.atlassian.net/browse/CUS-730|smart-link] )

!Screen Shot 2024-10-09 at 11.16.00 AM.png|width=988,height=374,alt="Screen Shot 2024-10-09 at 11.16.00 AM.png"!

*Text area*: clear field link

!Screen Shot 2024-10-09 at 11.18.21 AM.png|width=996,height=408,alt="Screen Shot 2024-10-09 at 11.18.21 AM.png"!

*Image select:* Image zoom buttons are not reachable via keyboard

!Screen Shot 2024-10-10 at 08.53.43 AM.png|width=940,height=758,alt="Screen Shot 2024-10-10 at 08.53.43 AM.png"!

*Time field:* Time select element cannot be reached via keyboard ([https://shopware.atlassian.net/browse/NEXT-39740|https://shopware.atlassian.net/browse/NEXT-39740|smart-link] )

!Screen Shot 2024-10-10 at 09.00.50 AM.png|width=948,height=368,alt="Screen Shot 2024-10-10 at 09.00.50 AM.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CUS-735**



[created from CUS-735, comment 501945]
